index.js

```diff
@@ -64,6 +65,24 @@ function generateV1Config (platformConfig) {
     return v1Config
 }
 
+function arrayMerge (objValue, srcValue, key) {
+    if (key === 'contentBlocking' && srcValue.exceptions) {
+        addExceptionsToUnprotected(srcValue.exceptions)
+    }
+
+    if (key === 'allowlistedTrackers') {
+        return mergeAllowlistedTrackers(objValue, srcValue)
+    }
+
+    if (Array.isArray(objValue)) {
+        return Array.from(new Set(objValue.concat(srcValue)))
+    }
+}
+
+function isFeatureMissingState (feature) {
+    return !('state' in feature)
+}
```

Comment:
when this can happen? Shouldn't that be caught by schema checking?

Reply:
Schema checking tests after build. Since there are features that we exempt the default state (and explicitly enable in the overrides) this is just a safe-guard to ensure the key exists.